integration/simple-cache-client.test.ts

```diff
@@ -579,3 +518,1513 @@ describe('Integration Tests for operations on sets datastructure', () => {
     expect(noKeyGetResponse).toBeInstanceOf(CacheSetFetch.Miss);
   });
 });
+
+describe('Integration tests for dictionary operations', () => {
+  it('should return InvalidArgument response for dictionaryGetField with invalid cache and dictionary names', async () => {
+    let response = await momento.dictionaryGetField(
```

Comment:
I realize that you got conflicting feedback on this on an earlier PR comment, and these are tests and not production code or public-facing examples so I don't need you to change them for this PR, but except in very rare cases I feel strongly that `let` is a code smell and should be avoided.  You do not need mutable variables here.

In the future if there is a question about changing an immutable variable to a mutable one please ping me so that we can discuss, I do not want to set a precedent for using unnecessary mutable variables and have that get perpetuated throughout the code base.  cc: @schwern .

I'd love to see these changed back but it does not need to block this PR or be handled urgently.